apps/admin-ui/src/styles/util.ts

```diff
@@ -147,7 +147,7 @@ export const InlineRowLink = styled(BasicLink).attrs({
 
 export const StyledHDSNavigation = styled(Navigation)`
   --breakpoint-xl: 9000px;
-  z-index: var(--tilavaraus-admin-stack-main-menu);
+  z-index: var(--tilavaraus-admin-stack-order-toast);
```

Comment:
Is this correct? it's the navigation, not the toast, and admin-stack-order-toast doesn't exist.
Could drop stack-main-menu to 100 (instead of 500) like --tilavaraus-stack-order-navigation in the ui.

Reply:
Right you are! Changed, and tested to work.